### PR TITLE
Update Readme File

### PR DESCRIPTION
**Pull request by mikeselander**
This PR is a first attempt to address some of the concerns raised in #91 and #132. It adds information concerning stylelint, adds a note about `installed_paths` errors, moves the Testing instructions into the CONTRIBUTING file and a few more improvements.

This could likely use some more love, I would love some thoughts on how we could make this even better!

### CODE REVIEW

**Comment by mikeselander** [comment]
@fklein-lu and/or @rmccue could I get some eyes on these updates?

File: readme.md [hunk 1]
```diff
@@ -218,3 +135,18 @@ While you will still have to manually install package peer dependencies, if you
 extends:
 - vendor/humanmade/coding-standards/packages/eslint-config-humanmade/.eslintrc
 ```
+
+## Using stylelint
+
+The stylelint package contains a [stylelint](https://stylelint.io/) configuration which you can use to validate your CSS and SCSS code style. We recommend you install and use eslint via npm directly or use [linter-bot](https://github.com/humanmade/linter-bot). See [the `@humanmade/stylelit` package README](packages/stylelint-config/readme.md) for more information on configuring stylelint to use the Human Made coding standards.
```
Comment (ntwb) [inline readme.md]:
```suggestion
The stylelint package contains a [stylelint](https://stylelint.io/) configuration which you can use to validate your CSS and SCSS code style. We recommend you install and use stylelint via npm directly or use [linter-bot](https://github.com/humanmade/linter-bot). See [the `@humanmade/stylelit` package README](packages/stylelint-config/readme.md) for more information on configuring stylelint to use the Human Made coding standards.
```

Comment (rmccue) [inline readme.md]: There's actually also another typo in the package name there 😬 

```suggestion
The stylelint package contains a [stylelint](https://stylelint.io/) configuration which you can use to validate your CSS and SCSS code style. We recommend you install and use stylelint via npm directly or use [linter-bot](https://github.com/humanmade/linter-bot). See [the `@humanmade/stylelint` package README](packages/stylelint-config/readme.md) for more information on configuring stylelint to use the Human Made coding standards.
```

Comment (mikeselander) [reply readme.md]: Ah, bugger. Good catch, both of you!

**Pull request merged**